File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_5/Q2.c
//WAP which accept no from user and print numbers till that number..

#include<stdio.h>


void Display(int iNo)
{
    int iCnt = 0;

     for ( iCnt = 1;  iCnt < iNo; iCnt++)
    {
        printf(" %d\t",iCnt);
    }
}



int main()
{
    int iValue =0;

    printf("ENTER NUMBER :");
    scanf("%d",&iValue);

    Display(iValue);


    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_20/Q2.java
//WAP to print numbers from N down to 1 in reverse order...

import java.util.Scanner;

class LogicX
{
   void sumEven(int iNo)
    {
        int iCnt = 0;
        

        for(iCnt = iNo; iCnt >= 1; iCnt--)
        {            
                 System.out.println(iCnt);
        }

    }
}

public class Q2
{
    public static void main(String A[])
    {


        Scanner sobj = new Scanner(System.in);

         int iNo=0;
      

        System.out.print("Enter Number : ");
         iNo = sobj.nextInt();


        LogicX lobj = new LogicX();

        

          lobj.sumEven(iNo);

    }
}
